src/JsonEditorWidget.php

```diff
@@ -137,7 +137,17 @@ public function run()
         // Prepare client options
         $clientOptions           = $this->clientOptions;
         $clientOptions['schema'] = $this->schema;
-        ArrayHelper::remove($clientOptions, 'startval');
+
+        try {
+            $parsedValue = Json::decode($this->value);
+        } catch (\Exception $e) {
+            $parsedValue = null;
```

Comment:
Why don't we show an error here?

Reply:
Right. Do we want to throw new exception or just display an error flashcard?

Reply:
You mean a flash message `Yii::$app->session->addFlash(...)` (error)?
I think so. CC: @handcode WDYT?

We need at least and **always** `Yii::error($e->getMessage(), __METHOD__)` in such cases in the catch block.

Reply:
I added the `Yii::error($e->getMessage(), __METHOD__)`